common/ioctl.edl

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) Open Enclave SDK contributors.
+// Licensed under the MIT License.
+
+/*
+**==============================================================================
+**
+** ioctl.edl:
+**
+**     This file declares OCALLs and ECALLs needed by the enclave to implement
+**     functions in POSIX ioctl.h
```

Comment:
Nit: Isn't ioctl.h a Linux header, not a POSIX one?

Reply:
ah you're right :smile: